### PR TITLE
Add webapi page types 7

### PR DESCRIPTION
**Pull request by wbamberg**
See  https://github.com/mdn/content/issues/16255. This is part 7 of a series of PRs to add page types to the Web/API documentation.

### CODE REVIEW

File: files/en-us/web/api/domstringlist/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: DOMStringList
 slug: Web/API/DOMStringList
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/domstringlist/index.md:4]: Strictly speaking this is not an interface, but we will remove it with the last `DOMString`, so this is ok for now.

File: files/en-us/web/api/ecdhkeyderiveparams/index.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: EcdhKeyDeriveParams
 slug: Web/API/EcdhKeyDeriveParams
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/ecdhkeyderiveparams/index.md:4]: Dictionary, but we decided to consider this as an interface, at least for now.

File: files/en-us/web/api/ecdsaparams/index.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: EcdsaParams
 slug: Web/API/EcdsaParams
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/ecdsaparams/index.md:4]: Dictionary

File: files/en-us/web/api/eckeygenparams/index.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: EcKeyGenParams
 slug: Web/API/EcKeyGenParams
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/eckeygenparams/index.md:4]: Dictionary

File: files/en-us/web/api/eckeyimportparams/index.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: EcKeyImportParams
 slug: Web/API/EcKeyImportParams
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/eckeyimportparams/index.md:4]: Dictionary

File: files/en-us/web/api/ecdsaparams/index.md (hunk 3 above)
Comment (wbamberg) [reply files/en-us/web/api/ecdsaparams/index.md:4]: I thought we had made the decision to treat these as interfaces?

**Review by teoli2003 — APPROVED** [review]
LGTM. I made some comments to track earlier decisions.

**Pull request merged**